packages/jsapi-shim/src/dh.ts

```diff
@@ -0,0 +1,24 @@
+// The Deephaven API script isn't packaged as a module (yet), and is just included in index.html, exported to the global namespace
+// This include file is simply a wrapper so that it behaves like a module, and can be mocked easily for unit tests.
+// https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#using-global-variables
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-console */
+
+declare global {
+  interface Window {
+    dh: any;
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, import/no-mutable-exports
+let dh: any;
+try {
+  dh = window.dh;
```

Comment:
`window` may not exist in SSR, did you try with `self` instead?

Reply:
I switched to `globalThis` instead. Available in all contexts (since Node 12 at least and in every modern browser for the last few years)

It's still possible for the API to be undefined, but at least the shim will just return undefined instead of crashing.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis